src/Servers/IIS/IIS/src/RequestRejectionReason.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Server.IIS
+{
+    internal enum RequestRejectionReason
```

Comment:
This was also copied (again I only use one value, so I can probably reduce it).

Reply:
Reduce it. It's internal.